packages/coffeescript/plugin/compile-coffeescript.js

```diff
@@ -83,8 +83,8 @@ export class CoffeeCompiler extends CachingCompiler {
 
     let sourceMap = JSON.parse(output.v3SourceMap);
 
-    if (source.match(/(`|import|export|function\*)/)) {
-      // If source contains backticks or ES2015+ features,
+    if (source.match(/`|\bimport\b|\bexport\b|\byield\b/)) {
```

Comment:
My vote would still be that, just for readability.  The back-tick at the end is harder to see and the difference is extremely negligible that that point. :smile: